torchci/pages/metrics.tsx

```diff
@@ -447,6 +468,10 @@ export default function Page() {
       value: stopTime,
     },
   ];
+  const timeParamsClickHouse = {
+    startTime: startTime.utc().format("YYYY-MM-DDTHH:mm:ss.SSS"),
+    stopTime: stopTime.utc().format("YYYY-MM-DDTHH:mm:ss.SSS"),
```

Comment:
Do you know if this is necessary?  Clickhouse ingestion handled typical iso format well

Reply:
I might do something wrong here.  Originally, I just use 

```
  const timeParamsClickHouse = {
    startTime: startTime,
    stopTime: stopTime),
  };
```

Which I like better because it's simple, the format looks like this `2024-08-12T21:31:12.269Z`, but ClickHouse doesn't like it and fails with the following error:

```
error - Error: Value 2024-08-12T21:31:12.269Z cannot be parsed as DateTime64(3) for query parameter 'startTime' because it isn't parsed completely: only 23 of 24 bytes was parsed: 2024-08-12T21:46:23.913.
```

Specifically, it doesn't like the Z (UTC) part.  I haven't been able to find a way to make ClickHouse accept that, trying to tweak different data type declaration, etc... https://clickhouse.com/docs/en/sql-reference/data-types/datetime64

Have you been able to parse a string like `2024-08-12T21:31:12.269Z` on ClickHouse?

Reply:
Oh interesting it doesn't like the "Z" at all.  I wonder what they do for ingestion then, since I don't think I do anything for that

Reply:
Ivan found https://clickhouse.com/docs/en/sql-reference/functions/type-conversion-functions#parsedatetimebesteffort which seems to handle iso format